### ISSUE TITLE
Adding Trakt Collections to Library.

### ISSUE THREAD

**Issue by mrXoo** [jurialmunkey/plugin.video.themoviedb.helper#405]
Hi There,

Im familiar with setting up libraries in kodi. Im trying to add my collections, both movies and tv to my library in kodi.

I have added a folder called movies and a folder called tvshows inaddon_data/ plugin.video.themoviedb.helper 

I even added 1 show and 1 movie to my library via the context menu but I can't get the whole collection to add.

Is there something im doing wrong?

**Comment by mathu204** [comment]
I could be wrong but I’m pretty sure jurial said somewhere before that he won’t be adding the option to add collections to library bc a trakt collection is meant to keep track of media you physically own, in which case the content in your collections should be added as a physical source(s) already in Kodi. Your best bet is making a new list (or two one for movies and one for shows) and copying all the movies and shows to the new list(s) and then add the list(s) to your library like that.

I did the above by making two trakt lists called My Movie Library, and My TV Library and added them to sync lists to library section in TMDBHelper so when I add a list to the movie or tv library list it gets added automatically when TMDBHelper runs an update. I also synced my watchlists.

TLDR: Basically, make new list(s) then add all your shows and movies, and the list(s) is now your new collection and should function as so.

**Comment by mrXoo** [comment]
It would be handy to be able to add your current trakt collection to the library as well, that way if you wanted to stream something from say netflix then you would be able to. Not all media can be actually owned on disk. Netflix is a great example of this.

**Comment by jurialmunkey** [comment]
I've covered this so many times before.

On Trakt, the collections feature is *only* for items that you physically own and thus makes absolutely no sense to sync to the library since it should already be in your library. If you have things in your collection that you don't own physically then you are misusing the collections feature.

Trakt provides custom user lists for content outside of your collection. Just create a separate list and add it.

**Comment by mathu204** [comment]
> It would be handy to be able to add your current trakt collection to the library as well, that way if you wanted to stream something from say netflix then you would be able to. Not all media can be actually owned on disk. Netflix is a great example of this.

You haven’t collected the content physically, it’s just from Netflix hence the name collection.

**Comment by mathu204** [comment]
> It would be handy to be able to add your current trakt collection to the library as well, that way if you wanted to stream something from say netflix then you would be able to. Not all media can be actually owned on disk. Netflix is a great example of this.

If you really want to add your Collection to Library u can use the Library Integration Tool it still works but not really for many addons. it does however work for TMDBHelper perfectly.

**Comment by TwilightMercy** [comment]
> > It would be handy to be able to add your current trakt collection to the library as well, that way if you wanted to stream something from say netflix then you would be able to. Not all media can be actually owned on disk. Netflix is a great example of this.
> 
> If you really want to add your Collection to Library u can use the Library Integration Tool it still works but not really for many addons. it does however work for TMDBHelper perfectly.

Does Library Integration Tool still work for Kodi 20? 
and with Fen?
